### ISSUE TITLE
Can work on lenovo ideapad 5 15IIL05 ?

### ISSUE THREAD

**Issue by Nking217** [RobyRew/Lenovo-Ideapad-Flex-5-14IlL05_Hackintosh_OpenCore#5]
just askin if it can work on this model.

**Specs:**
Processor
1x 10th Generation Intel® Core™ i5-1035G1 Processor(Core i5-1035G1)
Memory
1x 8GB DDR4 3200
Operating System
Windows 10 Home 64(HB:Hebrew,EN:English)
Hard Drive
1x 256GB PCIe NVMe
1x 1TB HDD 5400rpm
Wireless Network
1x Intel® Wi-Fi 6 AX201 2x2 AX; Bluetooth® 5.0 or above
Ports
1x HDMI 1.4b; SD Card reader; DC-in Jack (Round); Combo of 3.5mm Stereo Headphone Output; 1 Type C (USB 3.2 Gen 2 + DP 1.2 + PD 3.0); 2 USB 3.1 (Gen1)
Camera
1x 720P HD with Array Microphone & Privacy Shutter
Graphics
1x NVIDIA GeForce MX330 2GB
Monitor
15.6 FHD
Included Warranty
1CourierCarryin

**Comment by RobyRew** [comment]
> just askin if it can work on this model.
> 

Yes, just give me 1-2 weeks so i'll end classes, and after that I hope to update this

**Comment by Micael106** [comment]
HDMI work?

**Comment by RobyRew** [comment]
> just askin if it can work on this model.

Yes you should be able now with 0.7.8 on Big Sur for now
